howto/release.md

```diff
@@ -13,6 +13,9 @@ Make a release
   software in your main development switch, feel free to move to
   a fresh new switch to test this.)
 
+- instead of the previous, you can also run the fully automatic
+  'make test_install'
```

Comment:
could you explicitly mention that it rebuilds, installs, and run a test in a temporary directory?